### PR TITLE
[CoreRel] Allow for deprecated tests

### PR DESCRIPTION
**Pull request by gkastrinis**

### CODE REVIEW

File: src/testrel.jl [hunk 1]
```diff
@@ -164,6 +164,7 @@ Transaction Step used for `test_rel`
     results), then `broken` can be used to mark the tests as broken and prevent the test
     from failing.
   - `readonly`: If true, run the query as readonly
+  - `disable_corerel_deprecations::Bool`: control CoreRel deprecations
```
Comment (mmcgr) [inline src/testrel.jl:167]: Does the Step need a disable_corerel_deprecations flag? It seems more of a test_rel attribute.

Comment (gkastrinis) [reply src/testrel.jl]: I made a cleanup commit

File: src/testrel.jl [hunk 2]
```diff
@@ -398,7 +403,11 @@ function test_rel_steps(;
     end
 
     if abort_on_error
-        config_query *= """def insert[:rel, :config, :abort_on_error]: true\n"""
+        config_query *= """def insert(:rel, :config, :abort_on_error): true\n"""
+    end
+
+    if disable_corerel_deprecations
+        config_query *= """def insert[:rel, :config, :disable_corerel_deprecations]: "disable"\n"""
```
Comment (mmcgr) [inline src/testrel.jl:410]: Should this be
```suggestion
        config_query *= """def insert[:rel, :config, :corerel_deprecations]: "disable"\n"""
```

Comment (gkastrinis) [reply src/testrel.jl]: Oups! Fixed

**Review by mmcgr — APPROVED** [review]
I went ahead and successfully tested the effect in raicode, so I'm happy with this.

**Pull request merged**